src/dropdown.option.test.basics.ts

```diff
@@ -12,21 +12,6 @@ it('registers itself', async () => {
   );
 });
 
-it('is selectable', async () => {
-  const host = await fixture<GlideCoreDropdownOption>(
-    html`<glide-core-dropdown-option
-      label="Label"
-      selected
-    ></glide-core-dropdown-option>`,
-  );
-
-  const checkedIconContainer = host.shadowRoot?.querySelector(
-    '[data-test="checked-icon-container"] svg',
-  );
-
-  expect(checkedIconContainer?.checkVisibility()).to.be.true;
-});
-
```

Comment:
Replaced by [this](https://github.com/CrowdStrike/glide-core/pull/785/files#diff-d17d5fe675f698402dfd02b79bd80efef37b8b649e316915ec633fbbad515c31R171-R204) test below.